test/Dialect/Arc/make-tables.mlir

```diff
@@ -0,0 +1,35 @@
+// RUN: circt-opt %s --arc-make-tables | FileCheck %s
+
+// CHECK-LABEL: arc.define @Simple
+arc.define @Simple(%arg0: i4) -> i4 {
```

Comment:
Would be nice to have a test with multiple inputs and outputs and where the total input bits are 30 (to test for potential overflow issues).